product/application/searchService.go

```diff
@@ -48,21 +70,22 @@ func (s *ProductSearchService) Find(ctx context.Context, searchRequest *applicat
 	// pageSize can either be set in the request, or we use the configured default or if nothing set we rely on the ProductSearchService later
 	pageSize := searchRequest.PageSize
 	if pageSize == 0 {
-		pageSize = int(s.DefaultPageSize)
+		pageSize = int(s.defaultPageSize)
 	}
 
-	result, err := s.SearchService.Search(ctx, application.BuildFilters(*searchRequest, pageSize)...)
+	result, err := s.searchService.Search(ctx, application.BuildFilters(*searchRequest, pageSize)...)
 	if err != nil {
 		return nil, err
 	}
 
 	if searchRequest.PaginationConfig == nil {
-		searchRequest.PaginationConfig = s.PaginationInfoFactory.DefaultConfig
+		searchRequest.PaginationConfig = s.paginationInfoFactory.DefaultConfig
 	}
 
 	if pageSize != 0 {
 		if err := result.SearchMeta.ValidatePageSize(pageSize); err != nil {
-			s.Logger.WithContext(ctx).WithField("category", "application.ProductSearchService").Warn("The Searchservice seems to ignore pageSize Filter")
+			err = fmt.Errorf("the Searchservice seems to ignore pageSize filter, %w", err)
+			s.logger.WithContext(ctx).WithField(flamingo.LogKeySubCategory, "Find").Warn(err)
```

Comment:
Maybe `flamingo.LogKeyMethod` is more suitable here?

Reply:
The `flamingo.LogKeyMethod` key was originally intended for http methods on API call logs. We also have a lot of similar `flamingo.LogKeySubCategory` usages in the `CartService`

Reply:
Ah okay thanks for the clarification 👍🏻